src/Shared/Process/ProcessEx.cs

```diff
@@ -17,7 +17,7 @@ namespace Microsoft.AspNetCore.Internal
 {
     internal class ProcessEx : IDisposable
     {
-        private static readonly TimeSpan DefaultProcessTimeout = TimeSpan.FromMinutes(15);
+        private static readonly TimeSpan DefaultProcessTimeout = TimeSpan.FromHours(5);
```

Comment:
/fyi I _think_ this was irrelevant and the confusing messages were from https://github.com/dotnet/aspnetcore/pull/32889/files#diff-1bd55bf49e1eefafb35460c476d887d89b022be59b5c85c9e5509c67aa6ea0a7R55 using `DefaultProcessTimeout` instead of the `timeout` passed to the constructor. See also https://github.com/dotnet/aspnetcore/pull/32889/files#diff-1bd55bf49e1eefafb35460c476d887d89b022be59b5c85c9e5509c67aa6ea0a7R127

Reply:
Well that error isn't great :D

But the code path that failed isn't passing a timeout in https://github.com/dotnet/aspnetcore/blob/12401e7ed58a0690c9c510e5d80e9e4225aa5e47/src/ProjectTemplates/Shared/TemplatePackageInstaller.cs#L77

Reply:
Ah, I thought I added the specific timeout in the only place that mattered. This is a better hammer.

I opened #33057 for the error glitch.